packages/manager/core/generator/app/templates/package.json.hbs

```diff
@@ -24,11 +24,10 @@
     "@ovh-ux/manager-config": "^7.2.0",
     "@ovh-ux/manager-core-api": "*",
     "@ovh-ux/manager-react-core-application": "*",
+    "@ovh-ux/manager-tailwind-config": "*",
     "@ovh-ux/shell": "^3.0.0",
-    "@ovhcloud/msc-react-billing-tile": "*",
     "@tanstack/react-query": "^5.8.3",
     "@tanstack/react-query-devtools": "^5.8.3",
```

Comment:
The package `@ovhcloud/manager-components` is missing here too I think

Reply:
👍